content/docs/pulumi-cloud/access-management/scim/_index.md

```diff
@@ -24,6 +24,8 @@ The [Pulumi Cloud](https://app.pulumi.com) supports System for Cross-domain Iden
 If desired, in addition to the SCIM-managed teams, one can also configure and manage Pulumi-local teams in the Pulumi Cloud. See [Teams](/docs/pulumi-cloud/access-management/teams/) for how to configure teams in the Pulumi Cloud.
     {{% /notes %}}
 
+{{< sso-scim-limits-info idp="your Identity Provider" >}}
+
```

Comment:
In terms of visuals, it does look slightly strange to have 2 notes back to back with different styling. Would it be easy to make the SAML & SCIM notes consistent with the existing notes?

<img width="754" alt="Screenshot 2024-12-09 at 1 11 48 PM" src="https://github.com/user-attachments/assets/3cfc4801-984a-491e-aaeb-bd459ea610f0">

Reply:
I think I fixed this now:

![image](https://github.com/user-attachments/assets/23d96a2d-a1ac-4e1c-810d-9ad94d05e739)